chapter01/chapter01.adoc

```diff
@@ -0,0 +1,263 @@
+= Chapter 1: Introduction to MicroProfile
+
+== Introduction 
+
+This introductory chapter provides a comprehensive overview of the MicroProfile platform, setting the stage for subsequent chapters in this tutorial. It aims to familiarize you with the fundamentals of the MicroProfile platform, its need, and benefits. Finally, we will discuss its place in the broader context of enterprise Java development.
+
+== Topics to be covered:
+- What is MicroProfile
+- Need for MicroProfile
+- MicroProfile Specifications
+- Current MicroProfile Implementations
+- Architecture Philosophy
+- Benefits of MicroProfile
+- Relationship with Jakarta EE specification 
+
+== What is MicroProfile
+
+link:https://microprofile.io/[MicroProfile] is an open-source specification that facilitates the development of microservices using enterprise Java technologies. It defines a set of APIs and specifications for building modern, scalable, resilient, and efficient applications based on microservices architecture. The primary goal of MicroProfile is to simplify the development process for Java developers, enabling them to create applications optimized for cloud-native-development.
+
+A consortium of industry leaders, Java community members, and individual contributors collaborating within an open-source link:https://www.eclipse.org/org/workinggroups/about.php[Eclipse Foundation Working Group] developed MicroProfile. This collaborative approach ensured that MicroProfile was shaped by diverse experiences and requirements, making it a robust and versatile specification extending Java in the cloud-computing domain. 
+
+The MicroProfile Working Group currently comprises of following Java user groups and corporate members: 
+
+* Atlanta Java User Group (AJUG)
+* Association of the German Java User Groups (iJUG)
+* Garden State Java User Group (GSJUG)
+* Hunan AsiaInfo Anhui 
+* IBM 
+* Fujitsu
+* Red Hat
+* Primeton
+* Payara
+* Microsoft
+* Tomitribe
+
+== Need for MicroProfile
+
+The MicroProfile Specification was developed to address the following requirements:
+
+- *Microservices Architecture Adoption*: The industry shift towards microservices architecture offered several benefits like flexibility, scalability, and speed of deployment. At the same time, it has introduced several new challenges for developers due to the added complexities. These include ensuring seamless communication and integration between microservices, securing each microservice individually as well as interactions between them, managing performance and efficiency, designing microservices to be fault-tolerant and resilient to failures, ensuring data consistency across services, managing configurations across multiple environments and managing various independently deployable components. To address these challenges, MicroProfile provides a simplified and optimized set of APIs designed to build and deploy Java-based microservices applications.
```

Comment:
Consider mentioning Streaming APIs and possibly API first (Open API) in these feature highlights. I'd suggest mentioning eventual consistency (LRA) somewhere, but I'm not sure if that really 'rates' a highlight. I might be tempted to mention Reactive programming model as well.

Reply:
Fixed via commit c704f11